### PR TITLE
Task/01 setup

### PR DESCRIPTION
**Pull request by 3apy6a**
- [x]  Mentor is added to pr
- [x]  AC copied to pr description
- [x]  link to deployed version added to pr
- [x]  About block (text and image) added to the main page
- [x]  On the widths 390px and 1280px, the site layout corresponds to the design
- [x] Created separate component About
- [x]  Its styles added to [.module.css](https://github.com/css-modules/css-modules)
- [x]  For App.tsx styles moved to App.module.css

[лючшій сайт](https://3apy6a.github.io/fe-react-2024/)

### CODE REVIEW

File: src/components/about/About.component.tsx [hunk 1]
```diff
@@ -0,0 +1,20 @@
+import { TitleComponent } from '../title/Title.component';
+
+// @ts-ignore
```
Comment (vladyslav-yermolin-moc) [inline src/components/about/About.component.tsx]: Hey, @3apy6a, thanx for a good pr.
Please don't use `@ts-ignore` comment whenever you can.
It is the same as `!imortant` in CSS - only for the cases when you can't do the stuff without it.
Please try to refactor your code and remove ignore comment.

File: src/components/about/About.component.tsx [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+export const AboutComponent = ({ customClasses }) => (
+    <div className={customClasses}>
+        <TitleComponent>About me</TitleComponent>
+        <p>
+            {/* eslint-disable-next-line react/no-unescaped-entities */}
```
Comment (vladyslav-yermolin-moc) [inline src/components/about/About.component.tsx]: Same here. Just use entity, or use eslint autofix 
image

File: src/components/about/About.component.tsx [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+            Hi! My name is Susanna and I'm a Junior Frontend Developer. I am already familiar with main Web Technologies like React, HTML,
+            CSS, JavaScript and Git version control system.
+        </p>
+        {/* eslint-disable-next-line react/no-unescaped-entities */}
```
Comment (vladyslav-yermolin-moc) [inline src/components/about/About.component.tsx]: same here

File: src/index.css [hunk 4]
```diff
@@ -69,3 +70,68 @@ button:focus-visible {
     background-color: #F9F9F9;
   }
 }
+
+h2 {
+  font-size: 36px;
```
Comment (vladyslav-yermolin-moc) [inline src/index.css]: Please add a `line-height` here and for all texts

File: src/components/title/Title.component.tsx [hunk 5]
```diff
@@ -0,0 +1,2 @@
+// @ts-ignore
```
Comment (vladyslav-yermolin-moc) [inline src/components/title/Title.component.tsx]: Same for ignore

File: src/components/image/image.module.css [hunk 6]
```diff
@@ -0,0 +1,16 @@
+.img {
+  max-width: 100%;
+  border-radius: 20px;
```
Comment (vladyslav-yermolin-moc) [inline src/components/image/image.module.css]: Please correct according to design

File: src/index.css [hunk 7]
```diff
@@ -69,3 +70,68 @@ button:focus-visible {
...
+  font-weight: 700;
+  color: #FFF;
+}
+
+@media screen and (min-width: 390px) and (max-width: 736px) {
```
Comment (vladyslav-yermolin-moc) [inline src/index.css]: Please group all your media queries:
````
// mobile styles

// tablet styles
@media screen and (min-width: 390px) and (max-width: 736px) {
// all your styles for this width

}

File: src/components/image/Image.component.tsx [hunk 8]
```diff
@@ -0,0 +1,10 @@
+import portrait from '@/assets/portrait.png';
+
+import styles from './image.module.css';
+
+// @ts-ignore
```
Comment (vladyslav-yermolin-moc) [inline src/components/image/Image.component.tsx]: same for ignore

File: src/components/about/About.component.tsx [hunk 9]
```diff
@@ -0,0 +1,20 @@
...
+        <p>This page was developed during the course 'Intro to React' from Masters Academy in 2024.</p>
```
Comment (vladyslav-yermolin-moc) [inline src/components/about/About.component.tsx]: Please add a link for `'Intro to React'` according to design

File: src/components/about/About.component.tsx [hunk 10]
```diff
@@ -0,0 +1,20 @@
...
+        <p>
+            This is a social project from MOCG company where I got an opportunity to work with Frontend mentors and to create my own small
+            project for the portfolio.
+        </p>
+        <p>You can contact me via [social network name] and check out my GitHub.</p>
```
Comment (vladyslav-yermolin-moc) [inline src/components/about/About.component.tsx]: Please add a social network name and link (even a fake one)

Comment (vladyslav-yermolin-moc) [reply src/components/about/About.component.tsx]: same for GitHub

File: src/index.css [hunk 11]
```diff
@@ -69,3 +70,68 @@ button:focus-visible {
...
+    font-size: 19px;
+  }
+}
+
+p {
+  font-size: 22px;
+  color: #FFF;
+}
+
+@media screen and (min-width: 390px) and (max-width: 736px) {
```
Comment (vladyslav-yermolin-moc) [inline src/index.css]: If you remove `min-width`, you can fix this 
image

File: src/index.css [hunk 12]
```diff
@@ -69,3 +70,68 @@ button:focus-visible {
...
+h2 {
+  font-size: 36px;
+  font-weight: 700;
+  color: #FFF;
+}
+
+@media screen and (min-width: 390px) and (max-width: 736px) {
+
+  h2 {
+    font-size: 19px;
```
Comment (vladyslav-yermolin-moc) [inline src/index.css:79]: Please correct font-size, line-height and font-weight according to the design

File: src/components/about/About.component.tsx [hunk 13]
```diff
@@ -0,0 +1,20 @@
+import { TitleComponent } from '../title/Title.component';
+
+// @ts-ignore
+export const AboutComponent = ({ customClasses }) => (
+    <div className={customClasses}>
```
Comment (vladyslav-yermolin-moc) [inline src/components/about/About.component.tsx]:
```suggestion
    <article className={customClasses}>
```

File: src/index.css [hunk 14]
```diff
@@ -26,6 +26,7 @@ body {
   align-items: center;
   min-width: 320px;
   min-height: 100vh;
+  background: #000;
```
Comment (vladyslav-yermolin-moc) [inline src/index.css]: Please correct the bg-color according to design.
And remove unused styles for `:root`
```
 color-scheme: light dark;
  color: rgba(255, 255, 255, .87);
  background-color: #242424;
```

**Review by vladyslav-yermolin-moc — CHANGES_REQUESTED** [review]

File: src/components/about/About.component.tsx (hunk 1 above)
Comment (3apy6a) [reply src/components/about/About.component.tsx]: try to done

File: src/components/about/About.component.tsx (hunk 2 above)
Comment (3apy6a) [reply src/components/about/About.component.tsx]: done

File: src/components/about/About.component.tsx (hunk 3 above)
Comment (3apy6a) [reply src/components/about/About.component.tsx]: done

File: src/index.css (hunk 4 above)
Comment (3apy6a) [reply src/index.css]: done

File: src/components/title/Title.component.tsx (hunk 5 above)
Comment (3apy6a) [reply src/components/title/Title.component.tsx]: done

File: src/components/about/About.component.tsx (hunk 10 above)
Comment (3apy6a) [reply src/components/about/About.component.tsx]: done

File: src/index.css (hunk 11 above)
Comment (3apy6a) [reply src/index.css]: done

File: src/components/about/About.component.tsx (hunk 13 above)
Comment (3apy6a) [reply src/components/about/About.component.tsx]: done

File: src/index.css (hunk 12 above)
Comment (3apy6a) [reply src/index.css:79]: trying to make

File: src/index.css (hunk 14 above)
Comment (3apy6a) [reply src/index.css]: tried to do it

File: src/components/image/image.module.css (hunk 6 above)
Comment (vladyslav-yermolin-moc) [reply src/components/image/image.module.css]:
```suggestion
  border-radius: 40px;
```

File: src/index.css [hunk 15]
```diff
@@ -69,3 +68,72 @@ button:focus-visible {
...
+  }
+
+  .padding-mobile-none {
+    padding-top: 0;
+  }
+
+  .w-12 {
+    width: 100%;
+  }
+}
```
Comment (vladyslav-yermolin-moc) [inline src/index.css]:
```suggestion
p {
  font-size: 22px;
  color: #FFF;
  font-weight: 700;
  line-height: 32px;
}

section {
  padding: 40px;
}

.offset-right-xs {
  margin-right: 80px;
}

.wrapper {
  display: flex;
  flex-direction: row;
}

.w-6 {
  width: 50%;
}

.padding-120 {
  padding-top: 120px;
}

@media (max-width: 736px) {
 // all styles for mobile stored in one place
  h2 {
    font-size: 19px;
    line-height: 32px;
  }
  
  p {
    font-size: 16px;
    line-height: 19px;
  }

  .wrapper {
    flex-direction: column-reverse;
  }

  .offset-mobile-none {
    margin: 0;
  }

  .padding-mobile-none {
    padding-top: 0;
  }

  .w-12 {
    width: 100%;
  }
}
```

File: src/components/about/About.component.tsx (hunk 9 above)
Comment (vladyslav-yermolin-moc) [reply src/components/about/About.component.tsx]: Still here. And please add underline styles from design

File: src/components/about/About.component.tsx (hunk 10 above)
Comment (vladyslav-yermolin-moc) [reply src/components/about/About.component.tsx]: Please add underline styles from design

File: src/index.css [hunk 16]
```diff
@@ -1,7 +1,5 @@
 :root {
   font-family: Inter, system-ui, Avenir, Helvetica, Arial, sans-serif;
-  line-height: 1.5;
-  font-weight: 400;
   color-scheme: light dark;
   color: rgba(255, 255, 255, .87);
   background-color: #242424;
```
Comment (vladyslav-yermolin-moc) [inline src/index.css]:
```suggestion

```

Comment (3apy6a) [reply src/index.css]: done

File: src/components/about/About.component.tsx (hunk 10 above)
Comment (3apy6a) [reply src/components/about/About.component.tsx]: done

File: src/index.css (hunk 14 above)
Comment (3apy6a) [reply src/index.css]: done

File: src/index.css (hunk 15 above)
Comment (3apy6a) [reply src/index.css]: done

File: src/index.css (hunk 16 above)
Comment (3apy6a) [reply src/index.css]: done

File: src/index.css [hunk 17]
```diff
@@ -14,7 +9,7 @@
 a {
   font-weight: 500;
   color: #646CFF;
```
Comment (vladyslav-yermolin-moc) [inline src/index.css]: We can use for the links the font color of their parent bt [inherit](https://developer.mozilla.org/en-US/docs/Web/CSS/inherit)
```suggestion
  color: inherit;
```

As on design 
image

File: src/components/about/About.component.tsx [hunk 18]
```diff
@@ -0,0 +1,27 @@
...
+}
+
+export const AboutComponent: React.FC<AboutProps> = ({ customClasses }) => (
+    <article className={customClasses}>
+        <TitleComponent>About me</TitleComponent>
+        <p>
+            Hi! My name is Susanna and I&apos;m a Junior Frontend Developer. I am already familiar with main Web Technologies like React,
+            HTML, CSS, JavaScript and Git version control system.
+        </p>
+        <p>This page was developed during the course &apos;Intro to React&apos; from Masters Academy in 2024.</p>
```
Comment (vladyslav-yermolin-moc) [inline src/components/about/About.component.tsx:16]: I mean, on the design this is the link to our course promo. So we should add it here with [target="_blank"](https://developer.mozilla.org/en-US/docs/Web/SVG/Attribute/target) - to open in the new browser tab
```suggestion
        <p>This page was developed during the course <a href="https://www.mastersacademy.education/frontend-for-beginners-it" target="_blank" >&apos;Intro to React&apos;</a> from Masters Academy in 2024.</p>
```

File: src/index.css [hunk 19]
```diff
@@ -69,3 +65,82 @@ button:focus-visible {
...
+  font-weight: 700;
+  line-height: 32px;
+}
+
+p {
+  font-size: 22px;
+  color: #FFF;
+  font-weight: 700;
+  line-height: 32px;
+}
```
Comment (vladyslav-yermolin-moc) [inline src/index.css]: 🙃 I think, you can remove duplications and describe only one entity of each CSS [selector](https://developer.mozilla.org/en-US/docs/Learn/CSS/Building_blocks/Selectors)( class or tag). For whole this block `@media (max-width: 736px) {...}`
```suggestion
p {
  font-size: 22px;
  color: #FFF;
  font-weight: 700;
  line-height: 32px;
}

```

**Review by vladyslav-yermolin-moc — CHANGES_REQUESTED** [review]
Good job! You're so close 🤏

File: src/index.css (hunk 17 above)
Comment (3apy6a) [reply src/index.css]: mach better :)

File: src/components/about/About.component.tsx (hunk 18 above)
Comment (3apy6a) [reply src/components/about/About.component.tsx:16]: done

File: src/index.css (hunk 19 above)
Comment (3apy6a) [reply src/index.css]: 👍

File: src/index.css (hunk 17 above)
Comment (3apy6a) [reply src/index.css]: done

**Review by vladyslav-yermolin-moc — APPROVED** [review]
O'k, I merge this pr, but we have to sync

**Pull request merged**